src/main/ts/components/useData.ts

```diff
@@ -138,5 +149,7 @@ export default function useData(): UseDataReturn {
     caycStartingDate,
     chartStartingDate: chartDateRange[0],
     chartEndDate: chartDateRange[1],
+    selectedProjects: selectedProject,
+    setSelectedProjects: setSelectedProject,
```

Comment:
?

Reply:
`selectedProjects` is not needed here. I will remove.